Content.Shared/DeltaV/QuickPhrase/QuickPhrasePrototype.cs

```diff
@@ -0,0 +1,51 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Array;
+
+namespace Content.Shared.DeltaV.QuickPhrase;
+
+[Prototype("quickPhrase")]
+public sealed partial class QuickPhrasePrototype : IPrototype, IInheritingPrototype
+{
+    /// <summary>
+    /// The "in code name" of the object. Must be unique.
+    /// </summary>
+    [ViewVariables]
+    [IdDataField]
+    public string ID { get; private set; } = default!;
+
+    /// <summary>
+    /// The prototype we inherit from.
+    /// </summary>
+    [ViewVariables]
+    [ParentDataField(typeof(AbstractPrototypeIdArraySerializer<QuickPhrasePrototype>))]
+    public string[]? Parents { get; }
+
+    [ViewVariables]
+    [NeverPushInheritance]
+    [AbstractDataField]
+    public bool Abstract { get; }
+
+    /// <summary>
+    /// The phrase that this prototype represents.
+    /// </summary>
+    [DataField("text")]
```

Comment:
dont need `("text")` this since its the same as camelCased version of the field name
same for every other field